packages/components/Callout/macos/CalloutView.swift

```diff
@@ -6,7 +6,8 @@ class CalloutView: RCTView, CalloutWindowLifeCycleDelegate {
 
 	@objc public var target: NSNumber? {
 		didSet {
-			guard let targetView = bridge?.uiManager.view(forReactTag: target) else {
+			let targetView = bridge?.uiManager.view(forReactTag: target)
+			if (targetView == nil && target != nil) {
 				preconditionFailure("Invalid target react tag")
```

Comment:
Should we remove the "react tag" bit now and just say "invalid target"?

Reply:
sure